llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/pyproject.toml

```diff
@@ -26,7 +26,7 @@ dev = [
 
 [project]
 name = "llama-index-vector-stores-azureaisearch"
-version = "0.4.0"
+version = "0.4.1"
 description = "llama-index vector_stores azureaisearch integration"
 authors = [{name = "Your Name", email = "you@example.com"}]
```

Comment:
Forgot to add this last time, mind adding our names ? 
You can push your's and I'll commit mine ?